### PR TITLE
Release 1.18.1

### PR DESCRIPTION
**Pull request by ilya-zuyev**

### CODE REVIEW

**Review by medyagh — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # Bump these on release - and please check ISO_VERSION for correctness.
 VERSION_MAJOR ?= 1
 VERSION_MINOR ?= 18
-VERSION_BUILD ?= 0
+VERSION_BUILD ?= 1
```
Comment (spowelljr) [inline Makefile:18]: Bump the ISO version as well I believe

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (spowelljr) [reply Makefile:18]: Actually, with this comment I'm not sure...
```
# Default to .0 for higher cache hit rates, as build increments typically don't require new ISO versions
```

Comment (ilya-zuyev) [reply Makefile:18]: did we change anything in iso?

Comment (spowelljr) [reply Makefile:18]: I don't think so, I'm just not sure if we duplicate the v1.18.0 ISO and rename it to v1.18.1 or not

Comment (spowelljr) [reply Makefile:18]: Based on the comment I don't think we do

Comment (medyagh) [reply Makefile:18]: we didnt change ISO so no need to do that

**Review by spowelljr — APPROVED** [review]

**Pull request merged**